docs/htdocs/info/docs/tools/vep/script/vep_plugins.html

```diff
@@ -557,6 +602,23 @@ <h2 id="plugins_existing">Existing plugins</h2>
 mv G2P.pm ~/.vep/Plugins
 ./vep -i variations.vcf --plugin G2P,file=/path/to/G2P.csv
 </pre></p></p></div></td><td><div class="vdoc_dtype_count" style="white-space:normal;float:left;padding:2px 6px;cursor:default;background-color:#551177">Phenotype data and citations</div></td><td><ul style="padding-left:1em"><li><a href="https://metacpan.org/pod/List::Util" rel="external">List::Util</a> qw(any)</li><li><a href="https://metacpan.org/pod/Text::CSV" rel="external">Text::CSV</a></li><li><a href="https://metacpan.org/pod/Scalar::Util" rel="external">Scalar::Util</a> qw(looks_like_number)</li><li><a href="https://metacpan.org/pod/FileHandle" rel="external">FileHandle</a></li><li><a href="https://metacpan.org/pod/Cwd" rel="external">Cwd</a></li></ul></td><td>Ensembl</td></tr>
+
+<tr id="GeneBe" class="bg2 plugin_row" data-category="pathogenicity_predictions">
```

Comment:
Should the category be `Variant data` according to https://github.com/Ensembl/VEP_plugins/blob/release/112/plugin_config.txt ?

Reply:
Yes - please update the PR accordingly